mjpeg/__init__.py

```diff
@@ -66,6 +71,11 @@ def skip_data(stream, left):
         left -= len(rv)
 
 
+async def askip_data(stream: aiohttp.BodyPartReader, left: int):
+    buf = bytearray(range(left))
+    await read_data(buf, stream, left)
```

Comment:
Shouldn't you be awaiting aread_data() here instead read_data()?

Reply:
Wow, good catch!  Honestly, I have no idea why that hasn't prevented things from working in my tests tbh.

It may be that I haven't run into this code path in my tests, so I'll need to investigate after making that change